tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -69,7 +69,7 @@ typedef struct tiledb_array_schema_evolution_t tiledb_array_schema_evolution_t;
  */
 TILEDB_EXPORT int32_t tiledb_array_schema_evolution_alloc(
     tiledb_ctx_t* ctx,
-    tiledb_array_schema_evolution_t** array_schema_evolution);
+    tiledb_array_schema_evolution_t** array_schema_evolution) noexcept;
```

Comment:
There's no need for TILEDB_NOEXCEPT unless you to compile in C rather than C++. When such functions move into C visibility, they'll need to be changed.